AppInspector.CLI/Program.cs

```diff
@@ -25,103 +26,364 @@ static int Main(string[] args)
+                    throw new Exception(MsgHelp.GetString(MsgHelp.ID.ANALYZE_SIMPLETAGS_HTML_FORMAT));
+                }
+            }
+            else
+            {
+                CommonOutputChecks((CLICommandOptions)options);
+            }
+
+            return RunAnalyzeCommand(options);
+        }
+
+
+        /// <summary>
+        /// Checks that either output filepath is valid or console verbosity is not visible to ensure
+        /// some output can be achieved...other command specific inputs that are relevant to both CLI
+        /// and NuGet callers are checked by the commands themselves
+        /// </summary>
+        /// <param name="options"></param>
+        static void CommonOutputChecks(CLICommandOptions options)
+        {
+            //validate requested format
+            string fileFormatArgs = options.OutputFileFormat;
+            List<string> validFormats = new List<string>();
+            validFormats.Add("json");
+            validFormats.Add("text");
+
+            if (options is CLIAnalyzeCmdOptions)
+            {
+                fileFormatArgs = ((CLIAnalyzeCmdOptions)options).OutputFileFormat;
+            }
+
+            bool isValidFormat = validFormats.Any(v => v.Contains(fileFormatArgs));
+            if (!isValidFormat)
+            {
+                WriteOnce.Error(MsgHelp.FormatString(MsgHelp.ID.CMD_INVALID_ARG_VALUE, "-f"));
+                throw new OpException(MsgHelp.FormatString(MsgHelp.ID.CMD_INVALID_ARG_VALUE, "-f"));
+            }
+
+            //validate output is not empty if no file output specified
+            if (string.IsNullOrEmpty(options.OutputFilePath))
+            {
+                if (options.ConsoleVerbosityLevel.ToLower() == "none")
+                {
+                    WriteOnce.Error(MsgHelp.GetString(MsgHelp.ID.CMD_NO_OUTPUT));
+                    throw new Exception(MsgHelp.GetString(MsgHelp.ID.CMD_NO_OUTPUT));
+                }
+                else if (options.ConsoleVerbosityLevel.ToLower() == "low")
+                {
+                    WriteOnce.SafeLog("Verbosity set low.  Detailed output limited.", NLog.LogLevel.Info);
+                }
+            }
+            else
+            {
+                ValidFileWritePath(options.OutputFilePath);
+            }
+        }
+
+
+        /// <summary>
+        /// Ensure output file path can be written to
+        /// </summary>
+        /// <param name="filePath"></param>
+        static void ValidFileWritePath(string filePath)
+        {
+            try
+            {
+                File.WriteAllText(filePath, "");//verify ability to write to location
+            }
+            catch (Exception)
+            {
+                WriteOnce.Error(MsgHelp.FormatString(MsgHelp.ID.CMD_INVALID_FILE_OR_DIR, filePath));
+                throw new OpException(MsgHelp.FormatString(MsgHelp.ID.CMD_INVALID_FILE_OR_DIR, filePath));
+            }
+
+        }
+
```

Comment:
This is a lot of extra whitespace.

Reply:
Not seeing this was likely removed by the code clean up commit added after your review.

Reply:
I see two blank lines on 252 and 253 and 2 blank lines on 249 and 250. That's in addition to a blank line on 247 which is inconsistent with the style on 228/229.  I think you may need to double check what rules your code cleanup is applying because I don't think it is catching all the extra whitespace.